### PR TITLE
Updating license header

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: kratos/containers/bucket.h [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2008-03-12 14:20:36 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pooyan Dadvand
+//                   Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/containers/bucket.h:11]: I would say that for all of the files in the "containers" directoy the author is Pooyan, not me

File: kratos/geometries/geometry_data.h [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author:   JMCarbonell $
-//   Date:                $Date:   December 2015 $
-//   Revision:            $Revision:         1.7 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Josep Maria Carbonell
```
Comment (RiccardoRossi) [inline kratos/geometries/geometry_data.h]: this is Pooyan

File: kratos/includes/constant.h [hunk 3]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/constant.h:10]: pooyan

File: kratos/includes/dof.h [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/dof.h:10]: pooyan

File: kratos/includes/kernel.h [hunk 5]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/kernel.h:10]: pooyan

File: kratos/includes/kratos.h [hunk 6]
```diff
@@ -1,24 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/kratos.h:10]: pooyan

File: kratos/includes/process_info.h [hunk 7]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/process_info.h:10]: Pooyan

File: kratos/includes/serializer.h [hunk 8]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/includes/serializer.h:10]: Pooyan & Carlos

File: kratos/integration/integration_point.h [hunk 9]
```diff
@@ -1,12 +1,16 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author:   JMCarbonell $
-//   Date:                $Date:   December 2015 $
-//   Revision:            $Revision:         1.4 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Josep Maria Carbonell
```
Comment (RiccardoRossi) [inline kratos/integration/integration_point.h]: Pooyan

File: kratos/integration/quadrature.h [hunk 10]
```diff
@@ -1,9 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author:   JMCarbonell $
-//   Date:                $Date:   December 2015 $
-//   Revision:            $Revision:         1.7 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Josep Maria Carbonell
```
Comment (RiccardoRossi) [inline kratos/integration/quadrature.h]: Pooyan

File: kratos/linear_solvers/amgcl_ns_solver.h [hunk 11]
```diff
@@ -1,3 +1,17 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/amgcl_ns_solver.h]: Riccardo

File: kratos/linear_solvers/ilu0_preconditioner.h [hunk 12]
```diff
@@ -1,48 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/ilu0_preconditioner.h]: pooyan

File: kratos/linear_solvers/luc_solver.h [hunk 13]
```diff
@@ -1,50 +1,16 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.3 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/luc_solver.h]: pooyan

File: kratos/linear_solvers/power_iteration_eigenvalue_solver.h [hunk 14]
```diff
@@ -1,50 +1,16 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2009-01-15 11:11:35 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/power_iteration_eigenvalue_solver.h]: pooyan

File: kratos/linear_solvers/reorderer.h [hunk 15]
```diff
@@ -1,47 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/reorderer.h]: pooyan

File: kratos/linear_solvers/tfqmr_solver.h [hunk 16]
```diff
@@ -1,47 +1,15 @@
...
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/linear_solvers/tfqmr_solver.h]: pooyan

File: kratos/modeler/cartesian_mesh_generator_modeler.h [hunk 17]
```diff
@@ -1,47 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/modeler/cartesian_mesh_generator_modeler.h]: Jordi Cotela

File: kratos/modeler/edge_swapping_2d_modeler.h [hunk 18]
```diff
@@ -1,47 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/modeler/edge_swapping_2d_modeler.h]: pooyan

File: kratos/processes/calculate_embedded_signed_distance_to_3d_skin_process.h [hunk 19]
```diff
@@ -7,7 +7,8 @@
 //  License:		 BSD License
 //					 Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand, Ruben Zorrilla
+//  Main authors:    Pooyan Dadvand
+//                   Ruben Zorrilla
```
Comment (RiccardoRossi) [inline kratos/processes/calculate_embedded_signed_distance_to_3d_skin_process.h:11]: + Daniel Baumgaertner, Johannes Wolf

File: kratos/processes/calculate_signed_distance_to_3d_condition_skin_process.h [hunk 20]
```diff
@@ -1,48 +1,15 @@
...
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/processes/calculate_signed_distance_to_3d_condition_skin_process.h:10]: + Daniel Baumgaertner + Johannes Wolf

File: kratos/processes/graph_coloring_process.h [hunk 21]
```diff
@@ -1,50 +1,17 @@
...
+//                   Multi-Physics
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:33 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/processes/graph_coloring_process.h]: Pooyan

File: kratos/python/add_amgcl_solver_to_python.h [hunk 22]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_amgcl_solver_to_python.h]: Riccardo

File: kratos/python/add_banded_matrix_to_python.cpp [hunk 23]
```diff
@@ -1,24 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_banded_matrix_to_python.cpp:10]: Pooyan

File: kratos/python/add_compressed_matrix_to_python.cpp [hunk 24]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_compressed_matrix_to_python.cpp:10]: Pooyan

File: kratos/python/add_constant_matrix_to_python.cpp [hunk 25]
```diff
@@ -1,23 +1,16 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_constant_matrix_to_python.cpp:10]: pooyan

File: kratos/python/add_coordinate_matrix_to_python.cpp [hunk 26]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_coordinate_matrix_to_python.cpp:10]: pooyan

File: kratos/python/add_matrix_to_python.cpp [hunk 27]
```diff
@@ -1,24 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_matrix_to_python.cpp:10]: Pooyan

File: kratos/python/add_matrix_to_python.h [hunk 28]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_matrix_to_python.h:10]: Pooyan

File: kratos/python/add_mesh_to_python.h [hunk 29]
```diff
@@ -1,24 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_mesh_to_python.h:10]: Pooyan

File: kratos/python/add_model_part_to_python.h [hunk 30]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_model_part_to_python.h:10]: Pooyan

File: kratos/python/add_node_to_python.cpp [hunk 31]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_node_to_python.cpp:10]: Pooyan

File: kratos/python/add_parallel_linear_solvers_to_python.cpp [hunk 32]
```diff
@@ -1,23 +1,15 @@
-// Kratos Multi-Physics
```
Comment (RiccardoRossi) [inline kratos/python/add_parallel_linear_solvers_to_python.cpp:1]: this file would best go out

File: kratos/python/add_process_info_to_python.cpp [hunk 33]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_process_info_to_python.cpp:10]: pooyan

File: kratos/python/add_scalar_matrix_to_python.cpp [hunk 34]
```diff
@@ -1,24 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_scalar_matrix_to_python.cpp:10]: pooyan

File: kratos/python/add_serializer_to_python.cpp [hunk 35]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_serializer_to_python.cpp:10]: pooyan

File: kratos/python/add_serializer_to_python.h [hunk 36]
```diff
@@ -1,31 +1,13 @@
...
-
-//
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:34 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/python/add_serializer_to_python.h]: pooyan

File: kratos/python/add_sparse_matrix_to_python.cpp [hunk 37]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_sparse_matrix_to_python.cpp:10]: pooyan

File: kratos/python/add_sparse_vector_to_python.cpp [hunk 38]
```diff
@@ -1,23 +1,15 @@
...
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
+//                   Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/python/add_sparse_vector_to_python.cpp]: Pooyan

File: kratos/python/add_sparse_vector_to_python.h [hunk 39]
```diff
@@ -1,31 +1,13 @@
...
-
-//
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:34 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/python/add_sparse_vector_to_python.h]: Pooyan

File: kratos/python/add_symmetric_matrix_to_python.cpp [hunk 40]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_symmetric_matrix_to_python.cpp:10]: Pooyan

File: kratos/python/add_triangular_matrix_to_python.cpp [hunk 41]
```diff
@@ -1,23 +1,15 @@
...
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
 //
-// Copyright (c) 2016 Pooyan Dadvand, Riccardo Rossi, CIMNE (International Center for Numerical Methods in Engineering)
-// All rights reserved.
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
-// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/python/add_triangular_matrix_to_python.cpp:10]: pooyan

File: kratos/spatial_containers/bins_dynamic.h [hunk 42]
```diff
@@ -1,9 +1,13 @@
...
+//                   Multi-Physics
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: pooyan $
-//   Date:                $Date: 2007-03-27 17:02:19 $
-//   Revision:            $Revision: 1.1.1.1 $
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/spatial_containers/bins_dynamic.h]: in all spatial containers should be Pooyan and Carlos Roig

File: kratos/utilities/binbased_nodes_in_element_locator.h [hunk 43]
```diff
@@ -1,53 +1,18 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: pbecker $
-//   Date:                $Date: 2011-09-21 12:30:32 $
-//   Revision:            $Revision: 1.0 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pablo Becker
```
Comment (RiccardoRossi) [inline kratos/utilities/binbased_nodes_in_element_locator.h:10]: Riccardo, Pablo

File: kratos/utilities/body_distance_calculation_utils.h [hunk 44]
```diff
@@ -1,49 +1,17 @@
...
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Antonia Laresse
```
Comment (RiccardoRossi) [inline kratos/utilities/body_distance_calculation_utils.h:10]: riccardo ,Antonia

File: kratos/utilities/convect_particles_utilities.h [hunk 45]
```diff
@@ -1,54 +1,18 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:31 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Riccardo Rossi
```
Comment (RiccardoRossi) [inline kratos/utilities/convect_particles_utilities.h]: + Pablo

File: kratos/utilities/deflation_utils.h [hunk 46]
```diff
@@ -1,49 +1,16 @@
...
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/utilities/deflation_utils.h]: Riccardo

File: kratos/utilities/enrichment_utilities.h [hunk 47]
```diff
@@ -1,50 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: antonia $
-//   Date:                $Date: 2009-01-13 16:40:58 $
-//   Revision:            $Revision: 1.24 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Antonia Laresse
```
Comment (RiccardoRossi) [inline kratos/utilities/enrichment_utilities.h]: Riccardo, Pablo

File: kratos/utilities/enrichment_utilities_duplicate_dofs.h [hunk 48]
```diff
@@ -1,50 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: antonia $
-//   Date:                $Date: 2009-01-13 16:40:58 $
-//   Revision:            $Revision: 1.24 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Antonia Laresse
```
Comment (RiccardoRossi) [inline kratos/utilities/enrichment_utilities_duplicate_dofs.h]: pablo

File: kratos/utilities/kratos_log.cpp [hunk 49]
```diff
@@ -1,47 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: janosch $
-//   Date:                $Date: 2008-10-23 11:35:13 $
-//   Revision:            $Revision: 1.6 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    janosch
```
Comment (RiccardoRossi) [inline kratos/utilities/kratos_log.cpp]: pooyan

File: kratos/utilities/kratos_log.h [hunk 50]
```diff
@@ -1,47 +1,15 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: rrossi $
-//   Date:                $Date: 2007-03-06 10:30:34 $
-//   Revision:            $Revision: 1.2 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/utilities/kratos_log.h:10]: pooyan

File: kratos/utilities/split_tetrahedra_utilities.h [hunk 51]
```diff
@@ -1,50 +1,14 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: antonia $
-//   Date:                $Date: 2009-01-13 16:40:58 $
-//   Revision:            $Revision: 1.24 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Antonia Laresse
```
Comment (RiccardoRossi) [inline kratos/utilities/split_tetrahedra_utilities.h]: Riccardo

File: kratos/utilities/parallel_levelset_distance_calculator.h [hunk 52]
```diff
@@ -1,12 +1,16 @@
...
+//                   Multi-Physics 
 //
-//   Project Name:        Kratos
-//   Last Modified by:    $Author: pooyan $
-//   Date:                $Date: 2006-11-27 16:07:33 $
-//   Revision:            $Revision: 1.1.1.1 $
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
 //
+//  Main authors:    Pooyan Dadvand
```
Comment (RiccardoRossi) [inline kratos/utilities/parallel_levelset_distance_calculator.h]: Riccardo

**Review by RiccardoRossi — COMMENTED** [review]
i pointed out the authors in some cases in which it was obviously wrong. we'll iterate over this...

File: kratos/spatial_containers/bins_dynamic.h (hunk 42 above)
Comment (pooyan-dadvand) [reply kratos/spatial_containers/bins_dynamic.h]: This one is Nelson and carlos

**Comment by loumalouomega** [comment]
Done , @RiccardoRossi , sorry the previous headers lack a lot of information

**Review by RiccardoRossi — APPROVED** [review]

**Comment by rubenzorrilla** [comment]
I'm am merging this. Vicente is traveling right now.

**Pull request merged**